tests/integration/targets/connection_proxmox_pct_remote/aliases

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2024 Nils Stein (@mietzen) <github.nstein@mailbox.org>
+# Copyright (c) 2024 Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+azp/posix/3
+destructive
+needs/root
+needs/target/connection
+skip/docker
+skip/alpine
+skip/aix
+skip/osx
+skip/macos
+skip/freebsd
```

Comment:
I'd avoid skipping on the other platforms, or are there good reasons for skipping these?
```suggestion
skip/docker
skip/alpine
```

Reply:
> I'd avoid skipping on the other platforms, or are there good reasons for skipping these?

I skipped them since the target is GNU Linux and not BSD/Darwin. 
Btw. what is `aix`? Is this really this? https://en.wikipedia.org/wiki/IBM_AIX

They should run fine, but if they would fail, I would argue to **not** fix them. Since the fix would only be applied to fix **this** test and not a real world issue that would pop up on a GNU Linux system or proxmox.

Reply:
> I skipped them since the target is GNU Linux and not BSD/Darwin.

Not really. The plugin runs on any Unix like system (including WSL/WSL2). It usually connects by SSH to a Linux server, but the connection plugin itself can (and should) run on anything the ansible-core runs on.

>  Btw. what is `aix`? Is this really this? https://en.wikipedia.org/wiki/IBM_AIX

Yes. We had that in CI some time ago, but then it vanished. Before that I disabled it eventually (e528474054f76cfc9836e4774fe89e25f8649351) as it was too unstable. I guess we should remove the `skip/aix` entries at some point as they have been meaningless for a long time now...

Reply:
> Not really. The plugin runs on any Unix like system (including WSL/WSL2). It usually connects by SSH to a Linux server, but the connection plugin itself can (and should) run on anything the ansible-core runs on.

Ah OK now I your point, it is a bit confusing since this test is running on `ansible_host=localhost`, I just looked and found `ansible_host=ubuntu-2204` in `connection_incus`, does that mean that the `connection_incus` test connects to this VM: https://dev.azure.com/ansible/community.general/_build/results?buildId=133858&view=logs&j=cd6c16ea-aeec-5e0c-f215-cbc9bf57d902

Also what does:

```
non_local
unsupported
```

Mean in the aliases of `connection_incus`.

Reply:
`unsupported` basically means it doesn't run in CI, so it doesn't connects to any VM in CI :)

`non_local` has is for coverage reporting, apparently (you can find it in ansible-test's sources), but I don't think it has any effect since the test is marked as `unsupported` anyway...

> it is a bit confusing since this test is running on `ansible_host=localhost`.

It only uses `ansible_host=localhost` since we just need something to connect to by SSH to run the fake `pct` program :) A proper test would connect to a VM running `pct` (and likely also proxmox itself?), but we don't have the infrastructure for that.

Reply:
OK I thought I could check the OS with `ansible_distribution` but I get `'ansible_distribution' is undefined`. Can you give me a hint?